crates/ide_diagnostics/src/handlers/unresolved_module.rs

```diff
@@ -39,7 +39,7 @@ fn fixes(ctx: &DiagnosticsContext<'_>, d: &hir::UnresolvedModule) -> Option<Vec<
             .map(|candidate| {
                 fix(
                     "create_module",
-                    "Create module",
+                    &format!("Create module at {candidate}"),
```

Comment:
I don't feel strongly, but should we quote them?

```suggestion
                    &format!("Create module at `{candidate}`"),
```

Reply:
Ye that's a good point I think